Makefile

```diff
@@ -44,16 +44,12 @@ install::
 		echo "run 'make all' before you execute 'make install'!"; \
 		exit 1; \
 	fi
-	-mkdir -p "$(prefix)/bin"
-	-mkdir -p "$(prefix)/lib/mono/xbuild-frameworks"
-	-mkdir -p "$(prefix)/lib/xamarin.android"
-	-mkdir -p "$(prefix)/lib/mono/xbuild/Xamarin/"
+	-mkdir -p "$(prefix)/bin" "$(prefix)/lib/mono/xbuild-frameworks" \
```

Comment:
I also don't find it to be an improvement to go from one directory per line to requiring `\`.